kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/WiresPanelUi.java

```diff
@@ -468,6 +468,8 @@ public void onWireComponentCreated(WireComponent component) {
 
     @Override
     public void onWireComponentDeleted(WireComponent component) {
+        if (!WIRE_ASSET_PID.equals(component.getFactoryPid()))
+            configurations.deleteConfiguration(component.getPid());
```

Comment:
Hi, could you give me more context on this change?
Why we have 472 and 473 that are the same command?

Reply:
Sorry. My fault. 473 should be deleted.